src/locales/en.json

```diff
@@ -162,12 +162,16 @@
     "STACKS": "Stacks",
     "ACCOUNT_NAME": "Account",
     "RECEIVE": "Receive",
+    "CANCEL": "Cancel",
+    "REMOVE_WALLET": "Remove wallet",
     "SEND": "Send",
     "SWAP": "Swap",
     "BUY": "Buy",
     "MANAGE_TOKEN": "Manage token list",
     "STACKS_AND_TOKEN": "Stacks NFTs & SIP-10 tokens",
-    "ORDINALS": "Ordinals & BRC-20 Tokens"
+    "ORDINALS": "Ordinals & BRC-20 Tokens",
+    "REMOVE_FROM_LIST_TITLE": "Remove from list",
+    "REMOVE_FROM_LIST_DESCRIPTION": "This will remove this wallet from this list. You can import it again by connecting your device."
```

Comment:
Let's update this to: 

"This will disconnect and remove the wallet from Xverse. You can import the same wallet again later by reconnecting your device."